api/src/main/webapp/WEB-INF/queue.yaml

```diff
@@ -46,3 +46,15 @@ queue:
   retry_parameters:
     task_retry_limit: 1
     task_age_limit: 5m
+
+- name: createWorkspaceQueue
+  target: api
+
+  # rate parameters
+  bucket_size: 500
+  rate: 1/s
+  max_concurrent_requests: 10
+
+  retry_parameters:
+    task_retry_limit: 0
```

Comment:
`max_concurrent_requests: 10` seems low to me. It could cause e2e tests to take longer to finish. Also sometimes multiple CI e2e jobs are running concurrently.

Reply:
makes sense - I'll bump it to 100.